lib/turbolinks.rb

```diff
@@ -30,16 +31,11 @@ def referer
 
         require 'action_dispatch/routing/redirection'
         ActionDispatch::Routing::Redirect.class_eval do
-          def call_with_turbolinks(env)
-            status, headers, body = call_without_turbolinks(env)
-
-            if env['rack.session'] && env['HTTP_X_XHR_REFERER']
-              env['rack.session'][:_turbolinks_redirect_to] = headers['Location']
-            end
-
-            [status, headers, body]
+          if defined?(prepend) && Rails.version >= '4'
```

Comment:
Well, 3.2 doesn't support Ruby 2.2, so maybe we should exclude that from the build queue? So if that's the only failure I think we can get away with it.

This would be unlike #496 which failed on 3.2 everywhere: https://travis-ci.org/rails/turbolinks/jobs/55903649